### ISSUE TITLE
harmonize CHMO_0000953 (microscope) with OBI_0400169 (microscope) branch

### ISSUE THREAD

**Issue by StroemPhi** [rsc-ontologies/rsc-cmo#14]
I don't know which term was coined first but [obi:microscope](https://terminology.nfdi4chem.de/ts/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0400169&viewMode=All&siblings=false#) & [chmo:microscope](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000953&viewMode=All&siblings=false#) are clearly refering to the same thing and should thus be merged.

Also there is a [chmo:optical microscope](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000953&viewMode=All&siblings=false#) & [obi:optical microscope](https://terminology.nfdi4chem.de/ts/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0400169&viewMode=All&siblings=false#) which seem to be refering to the same thing.

**Comment by StroemPhi** [comment]
might be good to also keep an eye on what is happening wrt a more general "device" tree, see : https://github.com/obi-ontology/obi/issues/1467

**Comment by TobyAutoChem** [comment]
Also probably duplicate in to allotrope https://www.allotrope.org/ontologies

**Comment by StroemPhi** [comment]
> Also probably duplicate in to allotrope https://www.allotrope.org/ontologies
True: http://purl.allotrope.org/ontologies/equipment#AFE_0000825 is an exactMatch. However, it needs to be discussed how OBO and AFO are to be mapped.  IMO, this mapping should be done by AFO and not OBO, as AFO reuses alot from OBO by either directly importing it or by adapting it to its namespace. Whenever the latter is done, I would expect the devs to provide the source from which the adaptation was derived.
